src/sass/uikit-bootstrap/_elements.scss

```diff
@@ -440,6 +440,11 @@ a {
         background-color: #fff;
         border-color: $grey-border;
         color: $grey;
+        &:hover {
```

Comment:
All files in uikit-bootstrap are fetched from uikit-bootstrap project

And can be updated via a bash command

```bash build/update_uikit_bootstrap.sh```

Ref: https://github.com/govCMS/uikit-bootstrap/blob/master/sass/_elements.scss

Reply:
Hey @pandaskii this has been updated.